.buildkite/scripts/e2e-pipeline/README.md

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
E2E package is same with `main` branch script except:
- newly added `generate-steps.py`
- renames `STACK_VERSION` to `ELASTIC_STACK_VERSION` not to use another same env var